src/Interfaces.jl

```diff
@@ -495,8 +495,26 @@ function discover_parts_snd(
   parts_snd
 end
 
+function _warn_message_on_main_task_discover_parts_snd(data::AbstractPData)
+   map_parts(get_part_ids(data)) do part
```

Comment:
Use function `map_main` if you only want to map the task on the first task

Reply:
Ok. Good to know such function exists. Did not know. Solved now.